conans/client/conan_api.py

```diff
@@ -527,37 +532,37 @@ def info_build_order(self, reference, settings=None, options=None, env=None,
                          install_folder=None):
         reference, profile = self._info_get_profile(reference, install_folder, profile_name, settings,
                                                     options, env)
-
         recorder = ActionRecorder()
-        manager = self._init_manager(recorder)
-        graph = manager.info_build_order(reference, profile, build_order, remote_name, check_updates)
-        return graph
+        deps_graph, _, _ = self._graph_manager.load_graph(reference, None, profile, ["missing"], check_updates,
+                                                          False, remote_name, recorder, workspace=None)
+        return deps_graph.build_order(build_order)
 
     @api_method
     def info_nodes_to_build(self, reference, build_modes, settings=None, options=None, env=None,
                             profile_name=None, remote_name=None, check_updates=None, install_folder=None):
         reference, profile = self._info_get_profile(reference, install_folder, profile_name, settings,
                                                     options, env)
-
         recorder = ActionRecorder()
-        manager = self._init_manager(recorder)
-        ret = manager.info_nodes_to_build(reference, profile, build_modes, remote_name,
-                                          check_updates)
-        ref_list, project_reference = ret
-        return ref_list, project_reference
+        deps_graph, conanfile, _ = self._graph_manager.load_graph(reference, None, profile, build_modes, check_updates,
+                                                                  False, remote_name, recorder, workspace=None)
+        ret = []
+        for level in deps_graph.by_levels():
+            for node in level:
+                if node.binary == BINARY_BUILD:
+                    if node.conan_ref not in ret:
+                        ret.append(node.conan_ref)
```

Comment:
Shouldn't this block be in the graph manager too? feels a little like internal details.